app/helpers/govuk_design_system_helper.rb

```diff
@@ -58,4 +58,27 @@ def page_title(title, caption)
     caption_var = caption.strip.concat if caption.present?
     "#{caption_var} #{page_title_var} - #{service_name} - GOV.UK".strip
   end
+
+  def govuk_summary_list_entry(key, value, tag_options_key = {}, tag_options_value = {})
```

Comment:
I believe the new govuk_summary_list_entry, govuk_summary_key and  govuk_summary_value methods should all be public (to be consistent with govuk_detail,  govuk_breadcrumb_builder and govuk_page_title).

Reply:
Also, these will all require new tests in govuk_design_system_helper_spec.rb

Reply:
I have moved the methods to the public section and I've added spec tests.

Reply:
Apologies @MalcolmVonMoJ, my mistake, I have had another look at your changes and I think only the govuk_summary_list_entry should be public as this is the only method referenced from outside the class (in _court_applications.html.haml). Can govuk_summary_key and govuk_summary_value please be made private? (if they are public we should also have separate tests for them) Otherwise new test looks good.

Reply:
Done